Integrations/python/deephaven/java/primitives.py

```diff
@@ -1,16 +0,0 @@
-import jpy
-
-if jpy.has_jvm():
```

Comment:
I know I was not the only person to use this file.  I just borrowed something that already existed... beware that removing this may break others.

Reply:
I'm 100% sure this was not the code I thought it was. Not an issue.